example-clients/ipload.c

```diff
@@ -199,6 +234,9 @@ main (int argc, char *argv[])
 		}
 	}
 
+	if (autoclose_opt)
+		jack_deactivate(client);
+
```

Comment:
I was trying to follow the general code style, but it's very inconsistent...
I find that easy to read as-is, but that's me :)

I can add brackets, no problem.